### ISSUE TITLE
Limit amount data to sync

### ISSUE THREAD

**Issue by CaballeroAlexis** [sqlite-sync/SQLite-sync.com#8]
Hi, i want to synchronize in two nodes a table with a amount of 400000 rows. When i try it, the system never come back the JSON because the size of the data to send. There is not problem when y synchronize less info. 
So, i want to know if there is a way to limit the amount of data of the JSON. The only table i want to sync have the next columns:
Subscriber_id
Date
Name
RowVer
RowId
Thanks

**Comment by tdziemidowicz** [comment]
Do you need all 400 thousands rows on mobile? There is no simple way to split package of 400 thousands to smaller packages.
You can limit amount of rows that are sent to device using filters. See https://docs.sqlite-sync.com/#filters

**Comment by CaballeroAlexis** [comment]
Yes, i need all the rows in all the devices so the filter cant help me.
